_overviews/scala3-book/scala-for-javascript-devs.md

```diff
@@ -441,7 +441,7 @@ In both JavaScript and Scala, functions are objects, so their functionality is s
     </tr>
     <tr>
       <td class="scala-block">
-          <code>// technically this is a “method,” not a function
+          <code>// technically this is a “method”, not a function
```

Comment:
FWIW, it always looks funky to me, but the O’Reilly standard is to put the comma inside the closing quote. That being said, putting the comma after the closing quote looks better to me(!).

Reply:
Alas, the American "standard," such as it is, is to place commas and periods within quotes of all sorts, including definitional and "scare quotes," even though other punctuation goes outside unless the punctuation is being quoted...isn't this "lovely"?  (Contrast: He said, "yikes!".  I wasn't expecting him to say, "fine"!)

However, as language is a living thing (and the British style is already to put punctuation outside), I always recommend to use the more sensible formatting unless forced to do otherwise, and hope eventually it will grow to become the universal standard.

In this case, though, I wouldn't use quotes at all: `// technically, this is a method, not a function`

Reply:
I agree, that’s a nice change. (Thanks!)

Reply:
```suggestion
          <code>// technically this is a method, not a function
```